pages/flow/[id].js

```diff
@@ -83,6 +80,12 @@ export default function FlowPage({ asanas, currentFlowDB }) {
     });
   }
 
+  function handleMaxLength(text) {
+    if (text.length > 100) {
+      setDisabledSaveButton(true);
+    } else setDisabledSaveButton(false);
+  }
```

Comment:
You could shorten this a lot :)
```suggestion
  const handleMaxLength = (text) => setDisabledSaveButton(text.length > 100);
```

Reply:
Ah, nice one :) Thanks!